lib/api.js

```diff
@@ -498,3 +498,38 @@ exports.update_resources_access_mode_by_ids = function update_resources_access_m
 ) {
   return updateResourcesAccessMode(access_mode, "public_ids[]", ids, callback, options);
 };
+
+exports.create_metadata_field = function create_metadata_field(metadata, callback, options = {}) {
```

Comment:
Can you please change the callback to match v2 signature

Reply:
@strausr I am not sure what you mean. Could you explain this please?

Reply:
@strausr a possibly related question:
Should all the new functions be `1` or `2` in `lib/v2/api.js` and `lib/v2/uploader.js`? What does it mean?